build/Dockerfile

```diff
@@ -26,4 +26,4 @@ WORKDIR /app
 COPY --from=builder /app/bin/network-validator /usr/bin/network-validator
 COPY --from=builder /app/config/ /app/config/
 
-ENTRYPOINT network-validator --config=/app/config/config.yaml --timeout=1s
+ENTRYPOINT ["network-validator", "--config=/app/config/config.yaml"]
```

Comment:
This change was necessary to allow additional command-line options for the network-validator to be passed when running the docker image. This will require a new version of the docker image to be pushed.

Reply:
@boranx I think pushing a new version of the docker image is either something you've automated or handle manually? Could you PTAL?

Reply:
@tiwillia that's a nice change!, I've tested it on my local as well
We can locally produce the image via `make containerbuild` but it's not onboarded to _quay.io/app-sre_ yet
Afaik the current image is still hard-coded at https://github.com/openshift/osd-network-verifier/blob/main/pkg/cloudclient/aws/private.go#L50